bigtable/admin.go

```diff
@@ -275,6 +275,67 @@ func (ac *AdminClient) CreateColumnFamily(ctx context.Context, table, family str
 	return err
 }
 
+// set to true to make the table protected against data loss
+// i.e. deleting the table, the column families in the table,
+// and the instance containing the table would be prohibited
+type DeletionProtection int
+
+const (
+	UnSet DeletionProtection = 0
```

Comment:
NIT: "Unset" rather than "UnSet"

Reply:
I also think "True" and "False" could be confusing here. I would recommend the set "None, Protected, Unprotected".